### DISCUSSION TITLE
Have been following Patric's course

### DISCUSSION THREAD

**Discussion by ReyazBeigh** [Cyfrin/foundry-full-course-cu#3107] [General]
Just wondering why can't I send ETHs from the test networks. it says this error:

"You must hold at least 1 LINK on Ethereum Mainnet to request native tokens."

Is there a way around?

**Comment by EngrPips** [comment]
Hey @ReyazBeigh, Try this faucet https://cloud.google.com/application/web3/faucet/ethereum/sepolia. If it fails on your first try, you should wait and retry after 20 minutes.

**Reply by ReyazBeigh** [reply]
Thank you @EngrPips that worked!

**Reply by EngrPips** [reply]
Awesome, Keep crushing it, buddy!
